src/lib/operators/table_scan/abstract_table_scan_impl.hpp

```diff
@@ -24,20 +24,49 @@ class AbstractTableScanImpl {
    * @{
    */
 
-  template <bool CheckForNull, typename BinaryFunctor, typename LeftIterator>
+  template <bool CheckForNull, bool FunctorIsVectorizable, typename BinaryFunctor, typename LeftIterator>
```

Comment:
No, because clang would otherwise try to generate invalid code:

> Finally, a word on IsVectorizable: With the pragma giving the guarantee about no hidden dependencies, both gcc and clang can identify cases where SIMD is beneficial. However, when `loop vectorize` is set for clang, it throws errors for non-vectorized loops. To avoid these compiler errors, we don't even enter this method if we know that a certain iterator or functor cannot be vectorized.